dotnet/src/InternalUtilities/src/Data/VectorStoreRecordPropertyReader.cs

```diff
@@ -251,12 +260,89 @@ public static void VerifyPropertyTypes(List<PropertyInfo> properties, HashSet<Ty
     }
 
     /// <summary>
-    /// Get the serialized name of a property by first checking the <see cref="JsonPropertyNameAttribute"/> and then falling back to the property name.
+    /// Get the json property name of a property by using the <see cref="JsonPropertyNameAttribute"/> if available, otherwise
+    /// using the <see cref="JsonNamingPolicy"/> if available, otherwise falling back to the property name.
+    /// </summary>
+    /// <param name="options">The options used for json serialization.</param>
+    /// <param name="property">The property to retrieve a storage name for.</param>
+    /// <returns>The json storage property name.</returns>
```

Comment:
```suggestion
    /// Get the JSON property name of a property by using the <see cref="JsonPropertyNameAttribute"/> if available, otherwise
    /// using the <see cref="JsonNamingPolicy"/> if available, otherwise falling back to the property name.
    /// </summary>
    /// <param name="options">The options used for JSON serialization.</param>
    /// <param name="property">The property to retrieve a storage name for.</param>
    /// <returns>The JSON storage property name.</returns>
```

Reply:
Thanks, fixed.